src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -17,10 +28,792 @@ Automatic anomaly detection is the easiest way for you and your team to detect a
+
+  <Callout variant="tip">If you experience an error when assigning Slack channels, make sure that the New Relic AI Slack application has been [added to your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).</Callout>
+
+  3. Save the configuration.
+
+  You can modify the applications for each configuration at any time by selecting the configuration in the configuration table.
+
+### Use anomaly detection with webhooks:
+
+    1. Select **Webhook**.
+    2. Input the following information into the form:
+       * Provide the webhook URL.
+       * Provide optional custom headers.
+       * Choose to edit the custom payload, or enable using the default payload.
+    3. Save the configuration.
+
+      You can modify the applications for each configuration at any time by selecting the configuration in the configuration table.
+
+### Use anomaly detection with workflows: 
+
+  1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI** and click **Workflows** in the left pane.
+  2. Select **+ Add a workflow**.
+  3. Choose a name for your workflow that helps you easily identify it from others in your account.
+  3. Click the **Advanced** button to build an advanced filter for all attributes.
+  4. Click the **Select or enter attribute** selector and add `origins`. Then, select `contains` as the operator and `anomalies` as the value.
+  5. Select `entitiesData.types` `contains` `Application`.
+  6. Select `tag.entity.guid` to select an entity of your choice.
+  7. Select the `signalType` `contains` and the signal you would like to be notified on.
+  
+      <img
+      title="Configure your workflow"
+      alt="Configure your workflow"
+      src={anomalyWorkflowFilterData}
+      />
+
+  8. Click **Additional settings** if you want to enrich your data.
+  9. Select a destination channel of your choosing.
+  
+      <img
+      title="Notifications for workflows"
+      alt="A screenshot depicting destinations for workflows"
+      src={anomalyWorkflowNotify}
+      />
+
+  10. Click **Test workflow** if you want to check if you workflow works.
+  11. Click **Activate workflow**.
+
+  To reduce noise, we recommend sending anomalies to a low priority notification channel such as Slack or email.
+
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="mute-notifications"
+    title="Mute notifications"
+    >
+
+  In Slack, you can mute temporarily or permanently detections coming from specific applications. You can also mute temporarily the entire channel. This is useful in the case of an incident or when the channel should otherwise not be interrupted.
```

Comment:
```suggestion
  In Slack, you can temporarily or permanently mute detections coming from specific applications. You can also mute temporarily the entire channel. This is useful in the case of an incident or when the channel should otherwise not be interrupted.
```